service/controller/resource/deployment/master_deployment.go

```diff
@@ -120,7 +121,7 @@ func newMasterDeployments(customResource v1alpha1.KVMConfig, release *releasev1a
 							key.LabelCluster:      key.ClusterID(customResource),
 							key.LabelOrganization: key.ClusterCustomer(customResource),
 							"node":                masterNode.ID,
-							key.PodWatcherLabel:   key.OperatorName,
+							label.OperatorVersion: project.Version(),
```

Comment:
Just double checking -- this winds up being equivalent or not breaking others using this label?

Reply:
I reverted the change because the implications weren't clear.